api/auth.go

```diff
@@ -84,11 +83,11 @@ func (a *API) maybeLoadUserOrSession(ctx context.Context) (context.Context, erro
 	claims := getClaims(ctx)
 
 	if claims == nil {
-		return ctx, errors.New("invalid token")
+		return ctx, unauthorizedError("invalid token: missing claims")
 	}
 
 	if claims.Subject == "" {
-		return nil, errors.New("invalid claim: subject missing")
+		return nil, unauthorizedError("invalid claim: missing subject")
```

Comment:
Can we also change this to be `invalid claim: missing sub claim`. Furthermore, can we also try to parse the `claims.Subject` as UUID and return that information too `invalid claim: sub claim must be a UUID`.